capa/main.py

```diff
@@ -325,6 +339,15 @@ def pbar(s, *args, **kwargs):
     return matches, meta
 
 
+def find_capabilities(ruleset: RuleSet, extractor: FeatureExtractor, **kwargs) -> Tuple[MatchResults, Any]:
+    if isinstance(extractor, StaticFeatureExtractor):
+        extractor_: StaticFeatureExtractor = cast(StaticFeatureExtractor, extractor)
```

Comment:
the `isinstance(...)` type assertion *should* be enough for mypy to understand the types, but using `cast` might be even more explicit. i dont feel strongly that it should or shouldn't be present, just info.